docs/content/configuration/caching.md

```diff
@@ -24,6 +24,10 @@ for both broker and historical nodes, when defined in the common properties file
 
 #### Local Cache
 
+<div class="note caution">
+DEPRECATED: Use caffeine instead
```

Comment:
Can you please expand on this a little bit? Imagine a reader that doesn't know why it's deprecated, or when it will be removed, how urgent the switch is, and so on. Maybe something like:

> The local cache is deprecated in favor of the Caffeine cache, and may be removed in a future version of Druid. The Caffeine cache affords better performance and control over eviction behavior, and is recommended in any situation where you are using JRE 8u60 or higher.

Reply:
added

> The local cache is deprecated in favor of the Caffeine cache, and may be removed in a future version of Druid. The Caffeine cache affords significantly better performance and control over eviction behavior compared to `local` cache, and is recommended in any situation where you are using JRE 8u60 or higher.